__tests__/app.test.js

```diff
@@ -208,3 +209,44 @@ describe('GET /api/articles', () => {
 
 
 
+describe('POST /api/articles/:article_id/comments', () => {
+  it('should respond with the posted comment', () => {
+    const article_id = 1; // Use a valid article_id
+    const newComment = {
+      username: 'butter_bridge',
+      body: 'This is a test comment',
+    };
+
+    return request(app)
+      .post(`/api/articles/${article_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        const comment = res.body.comment;
+        expect(comment).toHaveProperty('comment_id');
+        expect(comment).toHaveProperty('votes', 0);
+        expect(comment).toHaveProperty('created_at');
+        expect(comment).toHaveProperty('author', newComment.username);
+        expect(comment).toHaveProperty('body', newComment.body);
+        expect(comment).toHaveProperty('article_id', article_id);
+      });
+  });
```

Comment:
Please write another test for a successful response to check if extra properties are ignored: 
```
const newComment = {
      username: 'butter_bridge',
      body: 'This is a test comment',
extraProperty: "someValue"
    }
```